### PR TITLE
Add new board tests

### PR DESCRIPTION
**Pull request by Herman0802**

### CODE REVIEW

File: tests/test_board.py [hunk 1]
```diff
@@ -1,60 +1,142 @@
 import unittest
+
 from rookpoly import Board
-from rookpoly import Polynomial
+from rookpoly.polynomial import Polynomial
```
Comment (DelevoXDG) [inline tests/test_board.py:4]: Why this change?

File: tests/test_board.py [hunk 2]
```diff
@@ -1,60 +1,142 @@
...
 
     def setUp(self):
         self.empty_boards = [Board(['0' * i for _ in range(i)]) for i in range(self.empty_boards_limit)]
-        self.board_1 = Board(["110", "100", "000", "001"])
-        self.board_2 = Board(["00111", "00111", "11100", "11100", "11000"])
+        self.board_global_1 = Board(["110", "100", "000", "001"])
+        self.board_global_2 = Board(["00111", "00111", "11100", "11100", "11000"])
+
+    def test_init(self):
+        # Testowanie poprawnej inicjalizacji planszy
```
Comment (DelevoXDG) [inline tests/test_board.py]: We should probably rewrite comments in English

File: tests/test_board.py [hunk 3]
```diff
@@ -1,60 +1,142 @@
...
+        self.assertEqual(self.board_global_2, ["00111", "00111", "11100", "11100", "11000"])
+
+    def test_remove_filled_rows(self):
+        # Testowanie usuwania wypełnionych wierszy z planszy
+        board = Board(["111", "000", "010"])
+        board.remove_filled_rows()
+        self.assertEqual(board, Board(["000", "010"]))
 
     def test_simplify(self):
+        # Testowanie uproszczenia planszy
```
Comment (DelevoXDG) [inline tests/test_board.py]: Maybe we should not add comments when the function name is self-explanatory

File: tests/test_board.py [hunk 4]
```diff
@@ -1,60 +1,142 @@
...
 
 class TestBoard(unittest.TestCase):
-
     empty_boards_limit = 7
 
     def setUp(self):
         self.empty_boards = [Board(['0' * i for _ in range(i)]) for i in range(self.empty_boards_limit)]
-        self.board_1 = Board(["110", "100", "000", "001"])
-        self.board_2 = Board(["00111", "00111", "11100", "11100", "11000"])
+        self.board_global_1 = Board(["110", "100", "000", "001"])
```
Comment (DelevoXDG) [inline tests/test_board.py]: Why the rename is needed?

File: tests/test_board.py [hunk 5]
```diff
@@ -1,60 +1,142 @@
...
+        # Testowanie identyfikacji pustych kwadratów na planszy
+        board = Board(["100", "001", "010"])
+        empty_squares = list(board.get_empty_squares())
+        self.assertEqual(empty_squares, [(0, 1), (0, 2), (1, 0), (1, 1), (2, 0), (2, 2)])
+
+    def test_get_set(self):
+        # Testowanie odczytu i zapisu wartości w komórkach planszy
+        board = Board(["000", "000", "000"])
+        board.set(1, 1, '1')
+        self.assertEqual(board.get(1, 1), '1')
```
Comment (DelevoXDG) [inline tests/test_board.py]: Let's get and check any other tile

File: tests/test_board.py [hunk 6]
```diff
@@ -1,60 +1,142 @@
...
+
+    def test_eq(self):
+        # Testowanie porównywania dwóch plansz
+        board_1 = Board(["100", "010", "001"])
+        board_2 = Board(["100", "010", "001"])
+        board_3 = Board(["101", "010", "001"])
+        self.assertTrue(board_1 == board_2)
+        self.assertFalse(board_1 == board_3)
+
+    def test_repr_str(self):
```
Comment (DelevoXDG) [inline tests/test_board.py]: Let's separate `repr` and `str` tests

File: tests/test_board.py (hunk 2 above)
Comment (DelevoXDG) [reply tests/test_board.py]: Reviewed prior to [b5cdb71](https://github.com/Herman0802/RookPoly/pull/3/commits/b5cdb7161e9e8f2428672f5e2ab6bdea3f9d9879)

File: tests/test_board.py (hunk 3 above)
Comment (Herman0802) [reply tests/test_board.py]: I think we need to take the same approach to all tests

File: tests/test_board.py (hunk 4 above)
Comment (Herman0802) [reply tests/test_board.py]: Because later in the tests local boards are also used

Comment (DelevoXDG) [reply tests/test_board.py]: This is a nitpick and you can leave it as is, but `self.board_1` would not be shadowing `board_1` anyway

File: tests/test_board.py (hunk 6 above)
Comment (Herman0802) [reply tests/test_board.py]: Okay why not

File: tests/test_board.py (hunk 1 above)
Comment (Herman0802) [reply tests/test_board.py:4]: For better import clarification

File: tests/test_board.py (hunk 5 above)
Comment (Herman0802) [reply tests/test_board.py]: Agree with you

File: tests/test_board.py (hunk 4 above)
Comment (Herman0802) [reply tests/test_board.py]: I understand, but since board data is declared before all tests, it is better to indicate their function in their name

Comment (DelevoXDG) [reply tests/test_board.py]: I understand the reasoning, but `self.` is indicative enough and global usually means something different in python.

Anyway, I'll leave it up to you

**Review by DelevoXDG — APPROVED** [review]
lgtm
